astropy/io/misc/asdf/tags/coordinates/tests/test_frames.py

```diff
@@ -0,0 +1,77 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+
+import pytest
+
+asdf = pytest.importorskip('asdf')
+from asdf.tests.helpers import assert_roundtrip_tree
+
+import astropy.units as u
+from astropy import units
+from astropy.coordinates import ICRS, FK5, Longitude, Latitude, Angle
+
+from ....extension import AstropyExtension
+
+
+def test_hcrs_basic(tmpdir):
+    ra = Longitude(25, unit=units.deg)
+    dec = Latitude(45, unit=units.deg)
+
+    tree = {'coord': ICRS(ra=ra, dec=dec)}
+
+    assert_roundtrip_tree(tree, tmpdir, extensions=AstropyExtension())
```

Comment:
Out of curiosity, is the `extensions` argument still needed? I thought it's registered as an entry point and we don't need to specify `extensions` any more.

Reply:
It should not be necessary.